torchbenchmark/models/yolov3/yolo_utils/torch_utils.py

```diff
@@ -25,6 +25,10 @@ def init_seeds(seed=0):
 
 def select_device(device='', apex=False, batch_size=None):
     # device = 'cpu' or '0' or '0,1,2,3'
+
+    if (torch.backends.mps.is_available()):
```

Comment:
This will skip a specified device if mps is available, is that intended?

Reply:
Good catch, i will add the CPU check.